icubam/db/test_synchronizer.py

```diff
@@ -217,7 +216,7 @@ def test_export_icus(self):
     with open("resources/test/icu2.csv") as csv_f:
       self.csv.sync_icus_from_csv(csv_f, False)
 
-    test_dir = tempfile.mkdtemp()
+    tempfile.mkdtemp()
```

Comment:
So visibly this temp folder is not actually used?

Reply:
good catch!